wot-wg-2023-draft.html

```diff
@@ -210,38 +210,50 @@ <h2>Scope</h2>
       <p>During this charter, the Working Group would work on the following:</p>
 
         <dl>
-          <dt><strong>Update Existing Specifications</strong>:</dt>
-          <dd>The WG will add new features and address issues discovered during implementation of already published specifications. 
-            Unlike the last charter, these would not be limited to backward-compatible updates.
+          <dt>
+		<strong>Support New Use Cases</strong>:
+          </dt>
+          <dd>Address the requirements of new use cases, such as Digital Twins, scalable events, 
+	    and interworking with edge and cloud systems.
+	    The WG intends to define canonical TDs, add a mechanism to sign TDs, 
```

Comment:
I do not understand why signing etc. are moved here.

Reply:
And it is part of the charter already. I do not understand how signing canonical TDs are part of a use case discussion that is comparable to digital twins. As the current charter says: 

> In other specifications, the WG intends to add a mechanism to sign TD

Reply:
Could move signing etc. under improving security, but I don't think it belongs under "new use cases".  It doesn't really address any specific requirement of a new use case, it improves security overall.